src/packages/core/extension-registry/models/file-upload-preview.model.ts

```diff
@@ -0,0 +1,7 @@
+import type { UmbFileUploadPreviewElement } from '../interfaces/file-upload-preview.interface.js';
+import type { ManifestElement } from '@umbraco-cms/backoffice/extension-api';
+
+export interface ManifestFileUploadPreview extends ManifestElement<UmbFileUploadPreviewElement> {
+	type: 'fileUploadPreview';
+	forMimeTypes: Array<string>;
```

Comment:
I believe we need to add some jsdocs to this property with examples on what to put into it.